stories/list/object/object-list.stories.js

```diff
@@ -0,0 +1,900 @@
+   <span class="fd-object-status fd-object-status--positive">
+   <span class="fd-object-status__text">Second status</span></span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Third Attribute</span>
+   </div>
+   </div>
+   </div>
+   </div> 
+  </li>
+  <li role="option" aria-selected="true" tabindex="0" class="fd-list__item is-selected">
+    <div class="fd-list__content">
+    <div class="fd-list-optional">
+    <span class="fd-list-optional__text">Optional inline text</span>
+    </div>
+    <div class="fd-list__object fd-list__object--top-row">
+    <span class="fd-avatar fd-avatar--s fd-avatar--circle fd-avatar--thumbnail" style="background-image: url('http://lorempixel.com/400/400/nature/4/')" role="presentation" aria-label="John Doe"></span>
+    <div class="fd-list__object fd-list__object--top-left">
+    <div class="fd-object-identifier">
+      <p class="fd-object-identifier__title">
+    Fitbit Versa Smart Watch, Black Fitbit Versa Smart Watch
+     </p>
+     </div></div>
+    <div class="fd-list__object fd-list__object--top-right fd-object-number fd-object-number--positive">
+    <span class="fd-object-number__text">956.00</span>
+    <span class="fd-object-number__unit">Euro</span>
+    </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom">
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-right">
+   <span class="fd-object-status fd-object-status--positive">
+   <span class="fd-object-status__text">First status</span></span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Second Attribute</span>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-right">
+   <span class="fd-object-status fd-object-status--positive">
+   <span class="fd-object-status__text">Second status</span></span>
+   </div>
+   </div>
+   <div class="fd-list__object fd-list__object--bottom-row">
+   <div class="fd-list__object fd-list__object--bottom-left">
+     <span class="fd-list-attribute">Third Attribute</span>
+   </div>
+   </div>
+   </div>
+   </div>
+  </li>
+</ul>
+</div>
+`;
+
+selection.storyName = 'Selection';
+
+selection.parameters = {
+    docs: {
+        iframeHeight: 305,
+        storyDescription: 'To create selection list use `fd-list--selection ` and `fd-list--row-selection` along with `fd-list and fd-list--object` modifier.'
+    }
+};
+
+export const multiSelection = () => `
+<h4 id="objectListItemMultiSelection">Object List Item With Multi Selection</h4>
+<div style="max-width: 450px">
+<ul class="fd-list fd-list--object fd-list--selection" role="listbox" aria-labelledby="objectListItem with multi selection">
```

Comment:
`aria-multiselectable="true"` should be added to multi selectable object list

```suggestion
<ul class="fd-list fd-list--object fd-list--selection" role="listbox" aria-multiselectable="true" aria-labelledby="objectListItemMultiSelection">
```

Reply:
updated